src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,17 @@
+.TodoInfo {
+  padding: 8px;
+  margin: 12px 0;
+  border: 1px solid #000;
```

Comment:
Consider using variables for border and background colors to avoid hardcoding. 🎨